src/Screens/signUp.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import { IMAGES, TOAST_MESSAGES } from '../constants';
+import '../Screens/signup.css';
+import { createUserWithEmailAndPassword } from 'firebase/auth';
+import { auth, db } from '../config/firebase.config';
+import { useNavigate } from 'react-router-dom';
+import { doc, setDoc } from 'firebase/firestore';
+import { toast } from 'react-toastify';
+import SignInWithGoogle from '../Components/SignInWithGoogle';
+
+const SignInPage = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState("");
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [error] = useState(null);
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await createUserWithEmailAndPassword(auth, email, password);
+      const user = auth.currentUser;
+      if(user){
+        await setDoc(doc(db,"Users",user.uid),{
+          email: user.email,
+          username: username,
+        });
+      }
+      //console.log("user registered successfully");
+      toast.success(TOAST_MESSAGES.SIGNUP_SUCCESS,{
+        position: "top-center",
+      });
+      navigate("/home");
```

Comment:
reuse the constant made in above comment here

Reply:
Done